Thu Sep 15 16:12:43 2022 - INFO  - Start to run test.
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]Thu Sep 15 16:12:44 2022 - ERROR - Failed in remote CMD operation:1
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]Thu Sep 15 16:12:47 2022 - ERROR - Failed in remote CMD operation:1
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]Thu Sep 15 16:12:50 2022 - ERROR - Failed in remote CMD operation:1
Traceback (most recent call last):
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen-riscv/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]Thu Sep 15 16:12:53 2022 - ERROR - Failed in remote CMD operation:1
Thu Sep 15 16:12:53 2022 - ERROR - oe_test_system_monitor_reboot.sh line 30
Thu Sep 15 16:12:54 2022 - WARN  - the remote password uses the default configuration.
Thu Sep 15 16:12:54 2022 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r @:/tmp/rebootlog2 .
ssh: Could not resolve hostname : Name or service not known
Thu Sep 15 16:12:55 2022 - ERROR - Failed in remote SCP operation: 1
Thu Sep 15 16:12:56 2022 - ERROR - oe_test_system_monitor_reboot.sh line 32
cat: rebootlog2: No such file or directory
Thu Sep 15 16:12:57 2022 - ERROR - oe_test_system_monitor_reboot.sh line 35
Thu Sep 15 16:12:57 2022 - INFO  - End to run test.
Thu Sep 15 16:12:58 2022 - ERROR - failed to execute the case.
Thu Sep 15 16:12:59 2022 - INFO  - Start to restore the test environment.
Thu Sep 15 16:12:59 2022 - INFO  - End to restore the test environment.
